src/grubconfigs.rs

```diff
@@ -152,13 +192,19 @@ mod tests {
     #[test]
     fn test_find_efi_vendordir() -> Result<()> {
         let td = tempfile::tempdir()?;
+
+        let td_updates = Path::new("/usr/lib/bootupd/updates/EFI");
```

Comment:
We can't do this in unit tests - it would attempt to change the host system (and since one is hopefully running the unit tests as as non-root, it would fail).

This is one reason I like the openat or newer cap-std approach because it makes it easy to in unit tests:

- Allocate a temporary directory
- Run some tests again that tempdir and know they're not accessing paths outside it

See e.g. https://github.com/containers/bootc/pull/417/commits/d46b072cd218ab114a01d604592b5c42cc1b8866#diff-323d27aaf67b976dd83fc169a9fba2ea73f8d7aba9587cabce7f9bab9a804885R465 which does this.